internal/pkg/cli/job_delete.go

```diff
@@ -275,6 +297,29 @@ func (o *deleteJobOpts) deleteStack(sess *awssession.Session, env string) error
 	return nil
 }
 
+func (o *deleteJobOpts) deleteTasks(sess *session.Session, env string) error {
+	cluster, err := o.newClusterGetter(sess).Cluster(o.appName, env)
+	if err != nil {
+		return fmt.Errorf("get cluster for env %s: %w", env, err)
+	}
+	ecsClient := o.newECSClient(sess)
+	tdFamilyName := fmt.Sprintf(ecs.FmtWorkloadTaskDefinitionFamily, o.appName, env, o.name)
+	tasks, err := ecsClient.FamilyTasks(cluster, tdFamilyName)
+	if err != nil {
+		return fmt.Errorf("list tasks that belong to family %s: %w", tdFamilyName, err)
+	}
+	var taskARNs []string
+	for _, task := range tasks {
+		taskARNs = append(taskARNs, *task.TaskArn)
```

Comment:
```
Although you can filter results based on a desired status of PENDING, this does not return any results. Amazon ECS never sets the desired status of a task to that value (only a task's lastStatus may have a value of PENDING).
```

I guess we can just filter by desired status to be RUNNING. What do you think?

Reply:
Maybe also `PENDING` because those will become `RUNNING` afterwards

Reply:
Hmm but based on what the doc says, the desired status just can't be PENDING because "Amazon ECS never sets the desired status of a task to that value (only a task's lastStatus may have a value of PENDING)."